src/form/dcp.jl

```diff
@@ -89,14 +89,14 @@ function add_setpoint_load!(sol, pm::_PMs.GenericPowerModel{T}) where T <: _PMs.
     mva_base = pm.data["baseMVA"]
     _PMs.add_setpoint!(sol, pm, "load", "pd", :z_demand; scale = (x,item,i) -> x*item["pd"][i])
     _PMs.add_setpoint_fixed!(sol, pm, "load", "qd")
-    _PMs.add_setpoint!(sol, pm, "load", "status", :z_demand, conductorless=true; default_value = (item) -> if (item["status"] == 0) 0 else 1 end)
+    _PMs.add_setpoint!(sol, pm, "load", "status", :z_demand; default_value = (item) -> if (item["status"] == 0) 0 else 1 end)
 end
 
 function add_setpoint_shunt!(sol, pm::_PMs.GenericPowerModel{T}) where T <: _PMs.AbstractDCPForm
     mva_base = pm.data["baseMVA"]
     _PMs.add_setpoint!(sol, pm, "shunt", "gs", :z_shunt; scale = (x,item,i) -> x*item["gs"][i])
     _PMs.add_setpoint_fixed!(sol, pm, "shunt", "bs")
-    _PMs.add_setpoint!(sol, pm, "shunt", "status", :z_shunt, conductorless=true; default_value = (item) -> if (item["status"] == 0) 0 else 1 end)
+    _PMs.add_setpoint!(sol, pm, "shunt", "status", :z_shunt; default_value = (item) -> if (item["status"] == 0) 0 else 1 end)
```

Comment:
Are you sure we can drop `conductorless=true` here?  May break a multi-conductor case.

Reply:
It was broken before, always giving the `default_value`. This change was needed to report the status from `z_shunt`.

Currently, some indicator variables are conductorless (`z_gen`) and others are !conductorless (`z_demand`, `z_shunt`).  Does it make sense to store these separately? i.e. have separate `z_demand` for each conductor?

Reply:
Interesting...

There should be a clear standard for all `z_` status variables and `conductorless` is the one that makes sense to me, but we need to make sure that the standard is working across all of the packages, especially PowerModelsDistribution.

I propose we leave this as you have it for now and merge this.  Then you, @pseudocubic, and I sit down on Monday afternoon for about an hour or two and make everything consistent across all of the packages.